dali/test/python/test_operator_multipaste.py

```diff
@@ -0,0 +1,218 @@
+    return lambda: source_list
+
+
+def get_pipeline(
+        batch_size=4,
+        in_size=None,
+        out_size=None,
+        even_paste_count=False,
+        k=4,
+        d_type=types.DALIDataType.UINT8,
+        no_intersections=True,
+        full_input=False,
+        in_anchor_top_left=False,
+        out_anchor_top_left=False
+):
+    pipe = Pipeline(batch_size=batch_size, num_threads=4, device_id=types.CPU_ONLY_DEVICE_ID)
+    with pipe:
+        input, _ = fn.file_reader(file_root=img_dir)
+        decoded = fn.image_decoder(input, device='cpu', output_type=types.RGB)
+        resized = fn.resize(decoded, resize_x=in_size[1], resize_y=in_size[0])
+        in_idx_l, in_anchors_l, shapes_l, out_anchors_l = prepare_cuts(
+            k, batch_size, in_size, out_size, even_paste_count,
+            no_intersections, full_input, in_anchor_top_left, out_anchor_top_left)
+        in_idx = fn.external_source(numpyize(in_idx_l))
+        in_anchors = fn.external_source(numpyize(in_anchors_l))
+        shapes = fn.external_source(numpyize(shapes_l))
+        out_anchors = fn.external_source(numpyize(out_anchors_l))
+        kwargs = {
+            "in_ids": in_idx,
+            "output_size": out_size,
+            "dtype": d_type
+        }
+
+        if not full_input:
+            kwargs["shapes"] = shapes
+
+        if not in_anchor_top_left:
+            kwargs["in_anchors"] = in_anchors
+
+        if not out_anchor_top_left:
+            kwargs["out_anchors"] = out_anchors
+
+        pasted = fn.multi_paste(resized, **kwargs)
+        pipe.set_outputs(pasted, resized)
+    return pipe, in_idx_l, in_anchors_l, shapes_l, out_anchors_l
+
+
+def manual_verify(batch_size, inp, output, in_idx_l, in_anchors_l, shapes_l, out_anchors_l, out_size_l):
+    for i in range(batch_size):
+        out = output.at(i)
+        out_size = out_size_l[i]
+        assert out.shape == out_size
+        ref = np.zeros(out.shape)
+        for j, idx in enumerate(in_idx_l[i]):
+            roi_start = in_anchors_l[i][j]
+            roi_end = roi_start + shapes_l[i][j]
+            out_start = out_anchors_l[i][j]
+            out_end = out_start + shapes_l[i][j]
+            ref[out_start[0]:out_end[0], out_start[1]:out_end[1]] = inp.at(idx)[roi_start[0]:roi_end[0],
+                                                                    roi_start[1]:roi_end[1]]
+        assert np.array_equal(out, ref)
+
+
+def show_images(batch_size, image_batch):
+    import matplotlib.gridspec as gridspec
+    import matplotlib.pyplot as plt
+    columns = 4
+    rows = (batch_size + 1) // (columns)
+    fig = plt.figure(figsize=(32, (32 // columns) * rows))
+    gs = gridspec.GridSpec(rows, columns)
+    for j in range(rows * columns):
+        plt.subplot(gs[j])
+        plt.axis("off")
+        plt.imshow(image_batch.at(j))
+    plt.show()
+
+
+def run_vs_python(bs, pastes, in_size, out_size, even_paste_count, no_intersections, full_input, in_anchor_top_left,
```

Comment:
```suggestion
def check_operator_multipaste(bs, pastes, in_size, out_size, even_paste_count, no_intersections, full_input, in_anchor_top_left,
```
Each test case will be printed when running all tests with nosetest if you follow also my suggestion below.